src/components/VersionSelect/index.spec.tsx

```diff
@@ -50,22 +54,22 @@ describe(__filename, () => {
   });
 
   it('renders two lists of versions', () => {
-    const listedVersions = [
+    const listedVersions: ExternalVersionsList = [
       {
```

Comment:
It would be more future proof to add `fakeVersionList` to `test-helpers` and extend it here, like:

```js
const listedVersions: ExternalVersionsList = [
  {
    ...fakeVersionList,
    id: 123,
    channel: 'listed',
    version: 'v1',
  },
]
```